bool is_confirmation
string robot_db
int16 block_id
